### PR TITLE
Add smart contract to style-guide

### PR DESCRIPTION
**Pull request by timhc22**
Have followed the format of the others

### CODE REVIEW

File: src/content/contributing/style-guide/index.md [hunk 1]
```diff
@@ -146,6 +146,19 @@ Proof-of-work should be capitalized at the beginning of a sentence. In any other
 
 The same rules we apply to proof-of-work are applicable to proof-of-stake, proof-of-authority, proof-of-humanity, proof-of-individuality, etc.
 
+### Smart Contract {#smart-contract}
```
Comment (minimalsm) [inline src/content/contributing/style-guide/index.md:149]: On ethereum.org we use title case for headings (except proper nouns :))

```suggestion
### Smart contract {#smart-contract}
```

File: src/content/contributing/style-guide/index.md [hunk 2]
```diff
@@ -146,6 +146,19 @@ Proof-of-work should be capitalized at the beginning of a sentence. In any other
...
+
+Smart contract should be capitalized at the beginning of a sentence. In any other instance, all letters should be lower case.
```
Comment (minimalsm) [inline src/content/contributing/style-guide/index.md:151]:
```suggestion
Smart contract is a common noun and should only be capitalized at the beginning of a sentence. In any other instance, all letters should be lowercase.
```

Comment (timhc22) [reply src/content/contributing/style-guide/index.md:151]: 'lower case' should be 'lowercase' on the 'Zero-knowledge' section also (as should be the 'is a common noun' part). Should I add those too, do I need to open another PR to do that, or can I do it on this one somehow?

**Comment by timhc22** [comment]
'lower case' should be 'lowercase' on the 'Zero-knowledge' section also (as should be the 'is a common noun' part). 

Should I add those changes too, or would it be better practice to open a separate PR?

**Review by minimalsm — APPROVED** [review]
Thanks @seb1220

**Pull request merged**